src/core/value.js

```diff
@@ -0,0 +1,134 @@
+/* @flow */
+
+'use strict';
+
+const GlobalBigNumber = require('bignumber.js');
+
+const BigNumber = GlobalBigNumber.another({
+  ROUNDING_MODE: GlobalBigNumber.ROUND_HALF_UP,
+  DECIMAL_PLACES: 40
+});
+
+class Value {
+
+  constructor(value: string | BigNumber, base: number) {
+    this._value = new BigNumber(value, base);
+  }
+
+  static getBNRoundDown() {
```

Comment:
Might be simpler to use class properties like the BigNumber class does. Basically just `Value.ROUND_DOWN = BigNumber.ROUND_DOWN` etc.